static/app/views/monitors/components/overviewTimeline/utils.tsx

```diff
@@ -0,0 +1,29 @@
+import moment from 'moment';
+
+import {
+  TimeWindow,
+  TimeWindowData,
+} from 'sentry/views/monitors/components/overviewTimeline/types';
```

Comment:
```suggestion
} from './types';
```

In this case relative imports are fine I think, since this stuff is all related to each other.

Reply:
tru tru, will change